index.js

```diff
@@ -47,7 +47,14 @@ var dateProperties = ['years', 'months', 'days']
 var timeProperties = ['hours', 'minutes', 'seconds']
 // according to ISO 8601
 PostgresInterval.prototype.toISO = function () {
-  var buildProperty = function (property) {
+  var datePart = dateProperties
```

Comment:
Can you also add line breaks between the two vars, the return, and the function declaration?